share/availability/full/reconstruction_test.go

```diff
@@ -112,7 +112,9 @@ func TestShareAvailable_ConnectedFullNodes(t *testing.T) {
 	})
 
 	// create light nodes and start sampling for them immediately
-	lights1, lights2 := make([]*availability_test.Node, lightNodes/2), make([]*availability_test.Node, lightNodes/2)
+	lights1, lights2 := make(
+		[]*availability_test.TestNode, lightNodes/2),
+		make([]*availability_test.TestNode, lightNodes/2)
```

Comment:
Nit: Personally for me it is much harder to read code as 3 line like that than if it was 2 lines:

```suggestion
	lights1 := make([]*availability_test.TestNode, lightNodes/2)
	lights2 := make([]*availability_test.TestNode, lightNodes/2)
```